Content.Server/Weapons/Ranged/Systems/GunSystem.cs

```diff
@@ -241,6 +243,7 @@ public void ShootProjectile(EntityUid uid, Vector2 direction, EntityUid? user =
         }
 
         Transform(uid).WorldRotation = direction.ToWorldAngle();
+        _throwing.TryThrow(physics.Owner, direction.Normalized * speed, user: user, pushbackRatio: 50f);
```

Comment:
Whatever you're trying to do this isn't the fix.

Reply:
then this will also be a problem for sprays like fire extinguisher and bottle sprays since they use the same way

Reply:
its better now